core/embed/rust/src/ui/layout_delizia/flow/receive.rs

```diff
@@ -18,13 +18,15 @@ use crate::{
         layout::util::ConfirmValueParams,
     },
 };
+use heapless::Vec;
 
 use super::super::{
     component::{AddressDetails, Frame, PromptScreen, SwipeContent, VerticalMenu},
     theme,
 };
 
 const QR_BORDER: i16 = 4;
+const MAX_XPUBS: usize = 3;
```

Comment:
Is this constant related to other `MAX_XPUBS` in other layouts?
Asking since they have different values:
```
core/embed/rust/src/ui/layout_bolt/component/address_details.rs:const MAX_XPUBS: usize = 16;
core/embed/rust/src/ui/layout_caesar/component/address_details.rs:const MAX_XPUBS: usize = 16;
core/embed/rust/src/ui/layout_delizia/component/address_details.rs:const MAX_XPUBS: usize = 16;
core/embed/rust/src/ui/layout_delizia/flow/receive.rs:const MAX_XPUBS: usize = 3;
core/embed/rust/src/ui/layout_eckhart/flow/receive.rs:const MAX_XPUBS: usize = 3;
```